mixer/pkg/runtime/routing/table.go

```diff
@@ -123,6 +143,24 @@ type InstanceGroup struct {
 	Mappers []template.OutputMapperFn
 }
 
+// OperationType enumeration
```

Comment:
nit: might be worth adding a bit more info here ;).

Reply:
yes, thanks.